client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/client/issue/ShowIssueParams.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.clientapi.client.issue;
+
+import java.util.List;
+import org.sonarsource.sonarlint.core.clientapi.common.FlowDto;
+import org.sonarsource.sonarlint.core.clientapi.common.TextRangeDto;
+
+public class ShowIssueParams {
+  private final String fileUri;
```

Comment:
I am not sure how usable a fileUri would be for us. I need to check. Also there is normally a mapping to do with paths because the server path and the IDE path can be different, not sure we can handle that in the backend today

Reply:
Renamed now to serverRelativeFilePath. The actual content is taken from the SQ response, eg _"src/main/java/com/github/kevinsawicki/http/HttpRequest.java"_